### DISCUSSION TITLE
Just a question that I don't know if it is possible with this new API

### DISCUSSION THREAD

**Discussion by SmilinStanSStanman** [albaintor/homeassistant_electrolux_status#11] [Ideas]
Hi all! Thanks for the resurrection of this integration.
There is a question I always wondered about this integration. In version V1 and now also in V2.
Could it be possible to rescue the text of the notifications that AEG/Electtrolux mobile apps send us and put them in a sensor or attribute?
There are questions that are not supported by actual sensors in my dishwasher. For example:
The rinse aid level entity is the qty of rinse aid that the dishwasher uses in every cycle. We don't know when we need to refill de rinse aid.
We have a similar problem with the salt. There is no way to know when we need to refill it.
If it is possible to catch the messages that the mobile app send us, we could make an automation when some messages appears to say us "hey, refill me!"

**Comment by tieskuh** [comment]
I don’t think that would be possible. Notifications are being sent via a different channel. Not via the API.

**Comment by albaintor** [comment]
No this is not possible, however this integration is richer than the previous one so you can trap the interesting sensors to build your notification
